azure_encryption_helper/encryptor.py

```diff
@@ -0,0 +1,47 @@
+from .aescipher import AESCipher
+from .aeskeywrapper import AESKeyWrapper
+
+import os
+
+class Encryptor:
+
+    def __init__(self, key=None, iv_length=16):
+        if key is None:
+            key = Encryptor.generate_aes_key()
+        self.key = key
+        self.iv_length = iv_length
+        self._update_cipher()
+
+    @staticmethod
+    def generate_aes_key(length=32):
+        return os.urandom(length)
+
+    def configure_wrapper(self, vault, credentials, key_name, key_version):
+        self.wrapper = AESKeyWrapper(vault, credentials, key_name, key_version)
+
+    def set_key(self, key, iv_length=None):
+        self.key = key
+        if iv_length is not None:
+            self.iv_length = iv_length
+        self._update_cipher()
+
+    def get_key(self, key):
+        return self.key
+
+    def set_wrapped_key(self, wrapped_key, iv_length=None):
```

Comment:
why not call this `unwrap_key`?

Reply:
imo unwrap_key assumes it returns unwrapped key.
So, we surely can do `encryptor.set_key(encryptor.unwrap_key(wrapped_key))`, but I don't see any other usage scenario for unwrap_key().. 

If you think we might need it, I can add this method.